stdlib/Sockets/src/Sockets.jl

```diff
@@ -22,6 +22,8 @@ export
     recv,
     recvfrom,
     send,
+    join_multicast_group,
+    leave_multicast_group,
```

Comment:
Nodejs calls these "addMembership/addSourceSpecificMembership" and "dropMembership/dropSourceSpecificMembership" (https://nodejs.org/dist/latest-v14.x/docs/api/dgram.html)

Reply:
Yeah, I see a lot of different names around.  I figured that these names were as self-explanatory as possible.